packages/astro/src/assets/services/sharp.ts

```diff
@@ -43,7 +50,11 @@ const sharpService: LocalImageService = {
 		// TODO: Sharp has some support for SVGs, we could probably support this once Sharp is the default and only service.
 		if (transform.format === 'svg') return { data: inputBuffer, format: 'svg' };
 
-		let result = sharp(inputBuffer, { failOnError: false, pages: -1 });
+		const result = sharp(inputBuffer, {
+			failOnError: false,
+			pages: -1,
+			limitInputPixels: config.service.config.limitInputPixels,
```

Comment:
Does Sharp handle it properly when this is `undefined`? The default for this is `268402689`, or `true`.

Reply:
Yeah I tested it locally, and when undefined (which is default) seems to still work fine like before. Not seeing any limit errors.

Reply:
The default is for limit errors to appear, what users are asking for is to be able to disable this

Reply:
Yeah if the image is too big that it hits the default, this configuration still works and would error. Also checked the source code and it seems to handle `undefined` fine:

https://github.com/lovell/sharp/blob/0499f59e710f8338779f7a29fec277a7b31da98f/lib/input.js#L40

https://github.com/lovell/sharp/blob/0499f59e710f8338779f7a29fec277a7b31da98f/lib/input.js#L113-L123

https://github.com/lovell/sharp/blob/0499f59e710f8338779f7a29fec277a7b31da98f/lib/is.js#L10-L12

Reply:
Ah okay, perfect then. Awesome work!